docs/variables/gamesplayed.md

```diff
@@ -0,0 +1,35 @@
+---
+id: gamesplayed
+---
+
+# $(gamesplayed)
+
+Returns a list of recorded directories that a broadcaster's latest stream was, or current stream is under.
+
+:::info This feature is managed by a custom tracking system!
+
+Fossabot only populates this list if it was part of your channel during at least one stream.
+
+If Fossabot has just joined your channel recently, there will likely be no recorded directories yet, even if you have already streamed in the past.
```

Comment:
*recorded games

to be consistent with the variable name and error msg

Reply:
I'll change it.